src/ErrorMessages.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Snackbar from '@material-ui/core/Snackbar';
+import { hideError } from './actions/errorActions';
+import SnackbarContentWrapper from './components/SnackbarContentWrapper';
+
+class ErrorMessages extends Component {
+  state = {
+    open: false
+  };
+
+  handleClose = (event, reason) => {
+    if (reason === 'clickaway') {
```

Comment:
Do you have more conditions like this? Maybe put them in a constants file would be a good idea

Reply:
That condition is from Material UI, and it's the only one that they mention